ArcGISRuntimeSDKQt_QMLSamples/CloudAndPortal/IntegratedWindowsAuthentication/IntegratedWindowsAuthentication.qml

```diff
@@ -0,0 +1,241 @@
+                        } else {
+                            webMapMsg.text = "Portal URL is empty. Please enter a portal URL"
+                            webMapMsg.visible = true;
+                            return;
+                        }
+                    }
+                }
+            }
+
+            ComboBox {
+                id:webmapsList
+                Layout.margins: 2
+                Layout.fillWidth: true
+                model: null
+            }
+
+            Button {
+                text: qsTr("Load Web Map")
+                Layout.fillWidth: true
+                Layout.margins: 2
+                onClicked: {
+                    if (webmapsList.currentIndex >= 0) {
+                        indicator.running = true;
+                        loadSelectedWebmap(portalItemListModel.get(webmapsList.currentIndex));
+                    }
+                }
+            }
+        }
+    }
+
+    BusyIndicator {
+        id: indicator
+        anchors.centerIn: parent
+        running: false
+    }
+
+    PortalQueryParametersForItems {
+        id: webmapQuery
+        types: [ Enums.PortalItemTypeWebMap ]
+    }
+
+    function searchPortal (portalUrl, forceLogin) {
+
+        var portal;
+        if (forceLogin) {
+            portal = ArcGISRuntimeEnvironment.createObject("Portal", {url: portalUrl, loginRequired: forceLogin});
+        } else {
+            portal = ArcGISRuntimeEnvironment.createObject("Portal", {url: portalUrl});
+        }
+
+        portal.loadStatusChanged.connect(function() {
+            if (portal.loadStatus === Enums.LoadStatusFailedToLoad) {
+                webMapMsg.text = portal.loadError.message;
+                webMapMsg.visible = true;
+                indicator.running = false;
+                return;
+            }
+
+            if (portal.loadStatus === Enums.LoadStatusLoaded){
+                portal.findItems(webmapQuery);
+                return;
+            }
+
+        });
+
+        portal.findItemsStatusChanged.connect(function() {
+            if ( portal.findItemsStatus === Enums.TaskStatusCompleted ) {
+                indicator.running = false;
+                portalItemListModel = portal.findItemsResult.itemResults;
+                var index = 0
+                var error = portalItemListModel.forEach(function(prtlItem) {
+                    listModelForComboBox[index] = prtlItem.title;
+                    index++;
+                });
+
+                if (error) {
+                    webMapMsg.text = error.message;
+                    webMapMsg.visible = true;
+                }
+                webmapsList.model = listModelForComboBox;
+            }
+        });
+
+        portal.load();
+        indicator.running = true;
+    }
+
+    function loadSelectedWebmap(selectedWebmap) {
+        portalItem = selectedWebmap;
+
+        portalItem.loadStatusChanged.connect(createMap);
```

Comment:
same comment as C++ -  no need to explicitly load PortalItem or Map. If you construct a Map and give it to the view, the view will kick off the load of the map which will load the portalitem. This should be a pretty simple function:

```
function loadSelectedWebmap(selectedWebmap) {
    mapView.map = ArcGISRuntimeEnvironment.createObject("Map", {"item": selectedWebmap});
}
```